CesiumGltf/include/CesiumGltf/MetadataConversions.h

```diff
@@ -340,6 +395,35 @@ template <> struct MetadataConversions<float, double> {
   }
 };
 
+/**
+ * @brief Converts from a std::string to a float.
+ */
+template <> struct MetadataConversions<float, std::string> {
+  /**
+   * @brief Converts a std::string to a float. This assumes that the entire
+   * std::string represents the number, not just a part of it.
+   *
+   * This returns std::nullopt if no number is parsed from the string.
+   *
+   * @param from The std::string to parse from.
+   */
+  static std::optional<float> convert(const std::string& from) {
+    if (from.size() == 0) {
+      // Return early. Otherwise, empty strings will be parsed as 0, which is
+      // misleading.
+      return std::nullopt;
+    }
+
+    char* pLastUsed;
+    float parsedValue = std::strtof(from.c_str(), &pLastUsed);
+    if (pLastUsed == from.c_str() + from.size() && !std::isinf(parsedValue)) {
```

Comment:
```suggestion
    if (errno == 0 && pLastUsed == from.c_str() + from.size() && !std::isinf(parsedValue)) {
```
I think we missed talking about this line from the last review. Would benefit from the same errno logic.